### PR TITLE
feat(frontend): Styled a navbar component with different UI for logge…

### PR DESCRIPTION
**Pull request by younome71**
Have designed the UI of the Nav-bar. 
Created a sign-in button for the users and also different UI for signed-in users and non-signed-in users.

### CODE REVIEW

File: frontend/google_icon.png
Comment (prajjwalkapoor) [inline frontend/google_icon.png:1]: Move this icon to the assets folder.

File: frontend/src/components/Navbar/index.tsx [hunk 1]
```diff
@@ -8,15 +8,44 @@ export default function Navbar() {
...
+      <Typography
+        variant="h1"
+        color={theme.palette.primary.main}
+        style={{fontSize: 60}}
+      >
         Busify
       </Typography>
-      <Typography>A centrailed bus ticket booking system</Typography>
+      <Typography style={{marginTop: '1.5rem', fontSize: 20}}>
+        A centrailed bus ticket booking system
```
Comment (prajjwalkapoor) [inline frontend/src/components/Navbar/index.tsx:26]: See Figma design, it is not there. Remove this.

File: frontend/src/components/Navbar/index.tsx [hunk 2]
```diff
@@ -8,15 +8,44 @@ export default function Navbar() {
...
+      </Typography>
       {isAuth ? (
-        <Typography>Hi! {user?.name}</Typography>
+        <div>
+          <Typography
+            style={{fontSize: 40, color: 'rgba(214, 202, 202, 0.788)'}}
```
Comment (prajjwalkapoor) [inline frontend/src/components/Navbar/index.tsx:31]: This should also be designed as per provided Figma UI.

File: frontend/src/components/Navbar/index.tsx [hunk 3]
```diff
@@ -8,15 +8,44 @@ export default function Navbar() {
...
+          <img
+            src=""
+            alt="User Image"
+            style={{width: 30, height: 30, borderRadius: 15}}
+          ></img>
+        </div>
       ) : (
-        <Button variant="contained" color="primary" href={getGoogleOAuthURL()}>
+        <Button
+          variant="contained"
```
Comment (prajjwalkapoor) [inline frontend/src/components/Navbar/index.tsx:43]: use outlined instead of this.

File: frontend/src/components/Navbar/index.tsx [hunk 4]
```diff
@@ -8,15 +8,44 @@ export default function Navbar() {
...
+          color="primary"
```
Comment (prajjwalkapoor) [inline frontend/src/components/Navbar/index.tsx:44]: it should be secondary.

File: frontend/src/components/Navbar/index.tsx [hunk 5]
```diff
@@ -8,15 +8,44 @@ export default function Navbar() {
...
+          href={getGoogleOAuthURL()}
+          style={{height: '60px', marginTop: '0.4rem'}}
+        >
+          <img src="google_icon.png" style={{height: '40px'}} />
```
Comment (prajjwalkapoor) [inline frontend/src/components/Navbar/index.tsx:48]: Don't add it here, pass it as a prop without any styling with the above button as `startIcon`.

File: frontend/src/components/Navbar/index.tsx [hunk 6]
```diff
@@ -8,15 +8,44 @@ export default function Navbar() {
...
+            style={{width: 30, height: 30, borderRadius: 15}}
+          ></img>
+        </div>
       ) : (
-        <Button variant="contained" color="primary" href={getGoogleOAuthURL()}>
+        <Button
+          variant="contained"
+          color="primary"
+          href={getGoogleOAuthURL()}
+          style={{height: '60px', marginTop: '0.4rem'}}
```
Comment (prajjwalkapoor) [inline frontend/src/components/Navbar/index.tsx:46]: remove these styles and add an 8px border-radius.

**Review by prajjwalkapoor — CHANGES_REQUESTED** [review]
@younome71 Do the required changes.

**Comment by prajjwalkapoor** [comment]
Also, there is one more button on the left side of the google signing button. Implement that also.

**Comment by prajjwalkapoor** [comment]
@younome71 Any updates?

**Comment by younome71** [comment]
@prajjwalkapoor Yes, It would be completed till 2-3 am tomorrow

**Pull request closed without merging**